lib/src/boilerplate_suggestors/boilerplate_utilities.dart

```diff
@@ -20,11 +20,40 @@ import 'package:over_react_codemod/src/util.dart';
 typedef YieldPatch = void Function(
     int startingOffset, int endingOffset, String replacement);
 
-@visibleForTesting
-bool isPublicForTest = false;
+SemverHelper semverHelper;
 
-// Stub while <https://jira.atl.workiva.net/browse/CPLAT-9308> is in progress
-bool isPublic(ClassDeclaration node) => isPublicForTest;
+/// Returns whether or not [node] is publicly exported.
+bool isPublic(ClassDeclaration node) {
+  assert(semverHelper != null);
+  return semverHelper.getPublicExportLocations(node) != null;
+}
+
+class SemverHelper {
+  Map _exportList;
+
+  SemverHelper(Map jsonReport) {
+    _exportList = jsonReport['exports'];
+  }
+
+  /// Returns semver report information for [node] if it is publicly exported.
+  ///
+  /// If [node] is not publicly exported, returns `null`.
+  Map<String, dynamic> getPublicExportLocations(ClassDeclaration node) {
```

Comment:
The name `getPublicExportLocations` implies it returns several locations, whereas this function only returns only a single result. I think that having multiple values would be helpful, so that we can tell the user all of the places a class is exported as opposed to just the first one we come accross.

Reply:
Also, the original AC had a signature of List<String>, where the `String` is the `key` of the `_exportLIst` map. Let's stick with that, since that's all the information we should need.